client/components/masonry-grid/docs/example.tsx

```diff
@@ -0,0 +1,36 @@
+import { PureComponent } from 'react';
+import { MasonryGrid } from 'calypso/components/masonry-grid';
+
+import './example.scss';
+
+export class MasonryGridExample extends PureComponent {
+	render() {
+		return (
```

Comment:
```suggestion
export function MasonryGridExample() {
```
No real reason to use a class component here, right?